### PR TITLE
Add multi-band MelGAN v2 config

### PR DESCRIPTION
**Pull request by kan-bayashi**
This PR adds new multi-band melgan config which using Adam without a gradient norm.
 
Related #143

### CODE REVIEW

**Comment by dathudeptrai** [comment]
@kan-bayashi in all my experiments, there are no different between batch_max_steps = 8192 and 16384 (include convergence speed). Maybe you can consider to use 8192 to speed-up training progress. I found model at 1M steps with 8192 and 16484 is the same (based on quality and loss) :))).

**Comment by kan-bayashi** [comment]
That is nice. I will try it.

**Comment by kan-bayashi** [comment]
Added comments about batch size.

**Comment by kan-bayashi** [comment]
batch_size=8192 is super fast. The training will be finished within 3 days:)

**Comment by dathudeptrai** [comment]
:)) that is why i run 4M steps with 4day on my 2080Ti :v

**Comment by azraelkuan** [comment]
can u share your tensorboard about melgan.v2, i found the D train too fast
image

**Comment by kan-bayashi** [comment]
Already uploaded.
I think if the discriminator loss keeps around 0.2 (not 0.0), it is OK.

**Comment by azraelkuan** [comment]
Thanks, my model works well after 600k

**Comment by azraelkuan** [comment]
I found your model's dis loss increase after 400k, at that steps, the disconnection disappear
